source/mapping-rules/mapping-rule-options/new-documents.txt

```diff
@@ -2,4 +2,83 @@
 
 =============
 New Documents
-=============
\ No newline at end of file
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+
+The :guilabel:`New Documents` mapping rule is the most basic mapping rule option
+which creates a document for each row of data in your relational database.
```

Comment:
(simple sentences)
Consider something like:
"New Documents is the most basic mapping rule option. It creates a document for each row of data in your relational database.

Reply:
Yes, good suggestion. Updated.